docs/src/content/docs/guides/overriding-components.mdx

```diff
@@ -81,7 +81,7 @@ When rendering a built-in component inside a custom component:
 
 - Spread `Astro.props` into it. This makes sure that it receives all the data it needs to render.
 - Add a [`<slot />`](https://docs.astro.build/en/core-concepts/astro-components/#slots) inside the default component. This makes sure that if the component is passed any child elements, Astro knows where to render them.
-
+- If the built-in component you are reusing also contains [named slots](https://docs.astro.build/en/basics/astro-components/#named-slots), you need to transfer them as well. For example, the [`TwoColumnContent`](/reference/overrides/#twocolumncontent) component uses a `right-sidebar` slot. To keep it you need to transfer it using the `<slot name="right-sidebar" slot="right-sidebar" />` syntax (see [transferring slots](https://docs.astro.build/en/basics/astro-components/#transferring-slots)).
```

Comment:
The ideas of the changes that were suggested during the Talking & Doc'ing event were:

- Explicitly mention the only 2 components that have named slots
- Show the example you initially provided with an highlight on the line transferring a slot


````suggestion

If you are reusing the [`PageFrame`](/reference/overrides/#pageframe) or [`TwoColumnContent`](/reference/overrides/#twocolumncontent) components which contain [named slots](https://docs.astro.build/en/basics/astro-components/#named-slots), you also need to [transfer](https://docs.astro.build/en/basics/astro-components/#transferring-slots) these slots as well.

The example below shows a custom component that reuses the `TwoColumnContent` component which contains an additional `right-sidebar` named slot that needs to be transferred:

```astro {9}
---
// src/components/CustomContent.astro
import type { Props } from '@astrojs/starlight/props';
import Default from '@astrojs/starlight/components/TwoColumnContent.astro';
---

<Default {...Astro.props}>
	<slot />
	<slot name="right-sidebar" slot="right-sidebar" />
</Default>
```

````